db/migrate/20160810095212_add_prefixes_to_find_your_local_council_transaction.rb

```diff
@@ -0,0 +1,13 @@
+class AddPrefixesToFindYourLocalCouncilTransaction < Mongoid::Migration
+  def self.up
+    find_your_local_council = Artefact.find_by_slug('find-your-local-council')
+    # Remove the exact route for /find-your-local-council...
+    find_your_local_council.paths = ['/find-your-local-council.json']
```

Comment:
dunno, just leaving it in there because it already was.  Maybe it'd be clearer to remove `/find-your-local-council` from paths instead of setting it to this?